framework/src/st_things/things_stack/things_types.h

```diff
@@ -253,13 +247,14 @@ typedef enum {
 } things_es_enrollee_state_e, things_prov_status_e, things_cloud_status_e;
 
 typedef struct access_point_info_s {
-	char e_ssid[MAX_SSID_LEN];		// mandatory
-	char security_key[MAX_SECUIRTYKEY_LEN];	// mandatory
+	char e_ssid[WIFIMGR_SSID_LEN];		// mandatory
```

Comment:
@harish-191 
Add +1 for NULL character.

Reply:
fixed.